keps/sig-apps/3189-downscale-pod-picker/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: Deployment/ReplicaSet Downscale Pod Picker
+kep-number: 3189
+authors:
+  - "@thesuperzapper"
+owning-sig: sig-apps
+participating-sigs: []
```

Comment:
You probably need to involve sig-architecture, as reaching out from the controller-manager into an external service is quite a change.

Reply:
I have added `sig-architecture` to `participating-sigs` in https://github.com/kubernetes/enhancements/commit/8beae1b5f2f9e7ed7814ce7a711c8633fe6f46d3